x-pack/legacy/plugins/siem/public/containers/detection_engine/rules/types.ts

```diff
@@ -5,6 +5,7 @@
  */
 
 import * as t from 'io-ts';
+import { ExportTimelineIds } from '../../../components/open_timeline/open_timeline';
```

Comment:
I think we can avoid imports from timeline here if we tried to do the suggestions on top.

Reply:
Agree, ExportTimelineIds has been removed.